target/linux/ramips/image/mt76x8.mk

```diff
@@ -79,6 +79,30 @@ define Device/buffalo_wcr-1166ds
 endef
 TARGET_DEVICES += buffalo_wcr-1166ds
 
+define Device/comfast_cf-wr758ac
+  IMAGE_SIZE := 7872k
+  DEVICE_VENDOR := Comfast
+  DEVICE_MODEL := WR758AC
+  DEVICE_ALT0_VENDOR := Joowin
+  DEVICE_ALT0_MODEL := WR758AC
+endef
```

Comment:
Additionally we might set `SUPPORTED_DEVICES` to the former model name for early users.

Reply:
Hello. Is it sufficient to set int in the MakeFile? Or do we also need to add it to the DTS file?

Reply:
In DTS files "compatible" strings are okay now, but the "model" strings should be changed as well.

Reply:
> 

Model strings in the DTS files fixed in another new push I just made.

So it's confirmed that there is no need to add joowin,jw-wr758ac-v{1,2} to the DTS files, I mean, it's enough to add the SUPPORTED_DEVICES entry in the makefile, correct?

Reply:
Precisely, the purpose of `SUPPORTED_DEVICES` is just for sysupgrade without force option.

Reply:
Thanks for the explanation.